### DISCUSSION TITLE
Is there anyway to customize System.Text.Json error message?

### DISCUSSION THREAD

**Discussion by mohammad-zr** [dotnet/runtime#78926] [Q&A]
https://github.com/dotnet/runtime/blob/129982ef61894433867f0ea593281fc069284ade/src/libraries/System.Text.Json/src/Resources/Strings.resx#L316

How can I customize message 'The JSON value could not be converted to...' inside an asp.net 7 api?

**Comment by ghost** [comment]
Tagging subscribers to this area: @dotnet/area-system-text-json, @gregsdennis
See info in [area-owners.md](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you want to be subscribed.
<details>
<summary>Issue Details</summary>
<hr />

https://github.com/dotnet/runtime/blob/129982ef61894433867f0ea593281fc069284ade/src/libraries/System.Text.Json/src/Resources/Strings.resx#L316

How can I customize message 'The JSON value could not be converted to...' inside an asp.net 7 api?

<table>
  <tr>
    <th align="left">Author:</th>
    <td>mohammad-zr</td>
  </tr>
  <tr>
    <th align="left">Assignees:</th>
    <td>-</td>
  </tr>
  <tr>
    <th align="left">Labels:</th>
    <td>

`area-System.Text.Json`, `untriaged`

</td>
  </tr>
  <tr>
    <th align="left">Milestone:</th>
    <td>-</td>
  </tr>
</table>
</details>

**Comment by Clockwork-Muse** [comment]
Normally, you don't.

If you want to return customer-meaningful error messages (such as for a web api), you'd need to install an error handler.

Why do you want to customize this message?
